diff --git a/script/build-libgit2-static.sh b/script/build-libgit2-static.sh
index 5723721..e23f2c1 100755
--- a/script/build-libgit2-static.sh
+++ b/script/build-libgit2-static.sh
@@ -14,6 +14,13 @@ cmake -DTHREADSAFE=ON \
       -DCMAKE_C_FLAGS=-fPIC \
       -DCMAKE_BUILD_TYPE="RelWithDebInfo" \
       -DCMAKE_INSTALL_PREFIX=../install \
+      -DUSE_EXT_HTTP_PARSER=OFF \
+      -DUSE_HTTPS=OFF \
+      -DUSE_NSEC=OFF \
+      -DUSE_SSH=OFF \
+      -DCURL=OFF \
+      -DUSE_GSSAPI=OFF \
+      -DUSE_BUNDLED_ZLIB=ON \
       .. &&
 
 cmake --build .
