diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4783e3ef9..d8f6378e1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,3 +1,4 @@
+
 # CMake build script for the libgit2 project
 #
 # Building (out of source build):
@@ -359,7 +360,6 @@ ENDIF()
 # Optional external dependency: libssh2
 IF (USE_SSH)
 	PKG_CHECK_MODULES(LIBSSH2 libssh2)
-ENDIF()
 IF (LIBSSH2_FOUND)
 	ADD_DEFINITIONS(-DGIT_SSH)
 	INCLUDE_DIRECTORIES(${LIBSSH2_INCLUDE_DIRS})
@@ -375,14 +375,15 @@ IF (LIBSSH2_FOUND)
 ELSE()
 	MESSAGE(STATUS "LIBSSH2 not found. Set CMAKE_PREFIX_PATH if it is installed outside of the default search path.")
 ENDIF()
+ENDIF()
 
 # Optional external dependency: libgssapi
 IF (USE_GSSAPI)
 	FIND_PACKAGE(GSSAPI)
-ENDIF()
 IF (GSSAPI_FOUND)
 	ADD_DEFINITIONS(-DGIT_GSSAPI)
 ENDIF()
+ENDIF()
 
 # Optional external dependency: iconv
 IF (USE_ICONV)
